gammapy/visualization/tests/test_utils.py

```diff
@@ -71,3 +73,35 @@ def test_plot_map_rgb():
     kwargs = {"stretch": 0.5, "Q": 1, "minimum": 0.15}
     with mpl_plot_check():
         plot_map_rgb(map_, **kwargs)
+
+
+def test_plot_distribution():
+    array = np.random.normal(0, 1, 10000)
+
+    array_2d = array.reshape(1, 100, 100)
+
+    energy_axis = MapAxis.from_energy_edges([1, 10] * u.TeV)
+
+    map_ = WcsNDMap.create(npix=(100, 100), axes=[energy_axis])
+    map_.data = array_2d
+
+    energy_axis_10 = MapAxis.from_energy_bounds(1 * u.TeV, 10 * u.TeV, 10)
+    map_empty = WcsNDMap.create(npix=(100, 100), axes=[energy_axis_10])
+
+    def fit_func(x, mu, sigma):
+        return norm.pdf(x, mu, sigma)
+
+    with mpl_plot_check():
+        res, axes = plot_distribution(
+            wcs_map=map_, func=fit_func, kwargs_hist={"bins": 40}
```

Comment:
Because of `curve_fit`, you need to define a python function that take the parameter to fit (here in the case of `norm` : `mu`and `sigma`), so that `curve_fit` knows which parameter to fit.